base/setup/lib/utils/arcname.c

```diff
@@ -226,7 +234,7 @@ ArcGetNextTokenU(
     if (!NT_SUCCESS(Status))
         return NULL;
 
-    TokenSpecifier->Length = wcslen(TokenSpecifier->Buffer) * sizeof(WCHAR);
+    TokenSpecifier->Length = (USHORT)SpecifierLength * sizeof(WCHAR);
```

Comment:
SpecifierLength is already a number of bytes, no need for the extra sizeof(WCHAR).

Reply:
Fixed